modules/log4j2/src/test/java/org/apache/ignite/logger/log4j2/Log4j2LoggerSelfTest.java

```diff
@@ -146,6 +146,11 @@ public void testSystemNodeId() throws Exception {
 
         assertEquals(U.id8(id), System.getProperty("nodeId"));
         assertEquals("other-app", System.getProperty("appId"));
+
+        new Log4J2Logger(LOG_PATH_TEST).setApplicationAndNode(null, id);
+
+        assertEquals(U.id8(id), System.getProperty("nodeId"));
```

Comment:
Did it change? Work without the fix

Reply:
Without the fix test fails with:
```
org.junit.ComparisonFailure: 
Expected :other-app
Actual   :ignite
```

Reply:
`assertEquals(U.id8(id), System.getProperty("nodeId"));` does not faild witout the fix. ok, might be checked too in same way.